### PR TITLE
Remove redundant front-end Checkout block test case

### PR DESCRIPTION
**Pull request by tarunvijwani**
We have one redundant [front-end test case file](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/frontend/checkout.test.js) for the Checkout block.

The `should display an empty cart message when cart is empty` test is already present in the [cart-checkout/checkout.test.js test file](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/shopper/cart-checkout/checkout.test.js).

In this PR we have: 

- Removed `should display an empty cart message when cart is empty` test case as we already had one in [cart-checkout/checkout.js.test](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/shopper/cart-checkout/checkout.test.js)
- Moved `allows customer to choose available payment methods` test to [cart-checkout/checkout.js.test](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/shopper/cart-checkout/checkout.test.js)
- Removed [frontend/checkout.test.js file](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/frontend/checkout.test.js) as all test cases are already covered in [cart-checkout/checkout.js.test](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/shopper/cart-checkout/checkout.test.js)

#### Automated Tests

* [x] Changes in this PR are covered by Automated Tests.
  * [ ] Unit tests
  * [x] E2E tests
 
### Manual Testing

How to test the changes in this Pull Request:

1. Check out this PR.
2. Start Docker Desktop.
3. Run `npm run wp-env destroy && npm run wp-env start` to start a fresh e2e environment.
4. Run `npm run test:e2e-dev -- tests/e2e/specs/shopper/cart-checkout/checkout.test.js` to run only this test case.
5. Make sure all tests pass.

### CODE REVIEW

File: tests/e2e/specs/shopper/cart-checkout/checkout.test.js [hunk 1]
```diff
@@ -53,6 +53,31 @@ describe( 'Shopper → Checkout', () => {
 			}
 		);
 	} );
+	describe( 'Shopper can change the payment method', () => {
```
Comment (alexflorisca) [inline tests/e2e/specs/shopper/cart-checkout/checkout.test.js]: There is only 1 test in this describe block so I'd suggest removing the `beforeAll` and `afterAll` hooks, and just calling `shopper.block.emptyCart()` at the start of the test.

I'd also suggest keeping to the current `describe` and `it` naming conventions. You can look at the other tests in this file for examples, but essentially the describe block should include the sub section we are testing (in this case "Payment Methods") and the `it` should be of the for "user can....."

**Review by alexflorisca — APPROVED** [review]
Looks good Tarun, just a quick suggestion for keeping things consistent

**Comment by tarunvijwani** [comment]
Thank you for reviewing this @alexflorisca! I have made the suggested changes. 👍

**Comment by alexflorisca** [comment]
Nice one, thanks Tarun, ship it! 🚢

**Pull request merged**